File: ruby_method/features/pages/login_page.rb
# encoding: utf-8

class LoginPage < SitePrism::Page
  set_url 'https://uat.ormuco.com/login'

  element  :select_language, 'div.lang_selected'
  elements :languages, '#lang_menu_dropdown td.lang_name'

  element  :main_logo, 'div.main_logo'

  element  :input_username, '#username'
  element  :input_password, '#password'
  element  :btn_forgot_password, '#login_form a'

  element  :warning_message, '#login_form div div div span'

  element  :btn_sign_in, '#login_form button'
  element  :btn_create_account, '#login_form > span'

  element  :btn_reseller_portal, 'div > a'
end

File: python_method/python_method/features/pages/loginPage.py
from page_objects import PageObject, PageElement, MultiPageElement

class LoginPage(PageObject):
    select_language = PageElement(css='div.lang_selected')

    languages = MultiPageElement(css='#lang_menu_dropdown td.lang_name')

    main_logo = PageElement(css='div.main_logo')

    input_username = PageElement(id_='username')
    input_password = PageElement(id_='password')
    btn_forgot_password = PageElement(css='#login_form a')

    warning_message = PageElement(css='#login_form div div div span')

    btn_sign_in = PageElement(css='#login_form button')
    btn_create_account = PageElement(css='#login_form > span')

    btn_reseller_portal = PageElement(css='div > a')
